funcx_web_service/routes/funcx.py

```diff
@@ -163,10 +163,10 @@ def auth_and_launch(
             user_id=user_id,
             function_id=function_uuid,
             container=container_uuid,
-            payload=payload,
+            # payload=payload,
```

Comment:
Can we delete this line rather than commenting it out?

```suggestion
```

Otherwise, we'll just have to cleanup later anyway. It's always possible to revert if we need to go back.

Reply:
Done